docs/content/docs/connectors/flink-sources/postgres-cdc.md

```diff
@@ -236,6 +236,15 @@ Connector Options
           so it does not need to be explicitly configured 'execution.checkpointing.checkpoints-after-tasks-finish.enabled' = 'true'
       </td>
     </tr>
+    <tr>
+      <td>scan.lsn-commit.checkpoints-num-delay</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">3</td>
+      <td>Integer</td>
+      <td>The number of checkpoint delays before starting to commit the LSN offsets. <br>
+          The checkpoint LSN offsets will be committed in rolling fashion, the earliest checkpoint identifier will be committed first from the delayed checkpoints.
```

Comment:
Could you explain to users why we need delay the offset commit?

Reply:
Sure, I think the previous version had a reason, I will add this:

```
This will enable continuous recycling of log files, preventing disk space issues.
```

Reply:
Done, please have a look @leonardBang, thanks